src/javascript/app/pages/user/metatrader/metatrader.ui.js

```diff
@@ -313,6 +313,21 @@ const MetaTraderUI = (() => {
                     $action.find('#frm_cashier').setVisibility(0);
                 }
             }
+
+            const remaining_transfers = getPropertyValue(State.getResponse('get_limits'), ['daily_transfers', 'mt5', 'available']);
+
+            if (typeof remaining_transfers !== 'undefined') {
+                const $remaining_container = $form.find('#mt5_remaining_transfers');
+                $remaining_container.setVisibility(1);
+                const $remaining_number = $remaining_container.find('strong');
+                $remaining_number.text(remaining_transfers);
+                if (+remaining_transfers) {
```

Comment:
may i know why?

Reply:
@mustofa-binary when the number reaches 0 we want to show it in red colour